17.json

```diff
@@ -0,0 +1,43 @@
+{
+  "mainTitle": "Payload Guest",
+  "creditsTitle": "Créditos",  
+
+  "loadingTick1": "Carregando.",
+  "loadingTick2": "Carregando..",
+  "loadingTick3": "Carregando...",
+
+  "ok": "Ok",
+  "select": "Selecionar",
+  "refresh": "Atualizar",
+  "credits": "Créditos",
+  "return": "Voltar",
+
+  "pageIndicator": "Página %i / %i", 
+  
+  "errorFatal": "ERRO FATAL!",
+  "errorCloseApplication": "Por favor, feche o Aplicativo.",
+  "errorLoadLibjbc": "Falha ao carregar o libjbc.prx", 
+  "errorInitializeLibjbc": "Falha ao inicializar libjbc",
+  "errorJailbreak": "Falha ao fazer jailbreak",
+
+  "errorPathOutput": "Local: %s",
+  "errorIndexOutput": "Índice: #%zu",
+  "errorHexOutput": "Erro: 0x%08x",
+
+  "errorMetadata": "Erro de Metadado",
+  "errorMetadataRead": "Falha ao ler o 'meta.json'",
+  "errorMetadataMalformed": "Malformação no 'meta.json'",
```

Comment:
The english translation uses the \` character, but in our language ` is a accent (à), So I thought it was kind of weird, then I changed it to '